main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -74,30 +77,42 @@ public static void main(String[] argv) {
       } else {
         gtfsInput = GtfsInput.createFromPath(Paths.get(args.getInput()));
       }
-    } catch (IOException | URISyntaxException | InterruptedException e) {
-      // FIXME: Add system errors to noticeContainer here.
+    } catch (IOException e) {
       logger.atSevere().withCause(e).log("Cannot load GTFS feed");
+      noticeContainer.addSystemError(new IOError(e.getMessage()));
+    } catch (URISyntaxException e) {
+      logger.atSevere().withCause(e).log("Syntax error in URL");
```

Comment:
Nit - if this is catching a `URISyntaxException`, the error message should say `URI` and not `URL`

Reply:
Thanks, I fixed the typo.